modules/huds/hudsAccesos.routes.ts

```diff
@@ -0,0 +1,42 @@
+import * as express from 'express';
+import { Auth } from '../../auth/auth.class';
+import { asyncHandler } from '@andes/api-tool';
+import { logAcceso } from './hudsAccesos';
+import { HudsAccesosCtr } from './hudsAccesos.controller';
+import { search } from './../../core/tm/controller/profesional';
+
+
+const router = express.Router();
+
+router.use(Auth.authenticate());
+router.get('/huds-accesos', asyncHandler(async (req: any, res) => {
+    const result = await HudsAccesosCtr.search(req.query);
+    res.json(result);
+}));
+
+router.post('/huds-accesos/huds-token', asyncHandler(async (req: any, res) => {
```

Comment:
los huds son redundantes
```suggestion
router.post('/accesos/token', asyncHandler(async (req: any, res) => {
```